Lib/test/test_glob.py

```diff
@@ -513,7 +513,14 @@ def fn(pat):
             return glob.translate(pat, recursive=True, include_hidden=True, seps=['/', '\\'])
         self.assertEqual(fn('foo/bar\\baz'), r'(?s:foo[/\\]bar[/\\]baz)\Z')
```

Comment:
More generally, can you upodate `test_translate_matching` and include the examples of https://man7.org/linux/man-pages/man7/glob.7.html so that we have a compliant implementation?